docs/core_docs/docs/tutorials/chatbot.ipynb

```diff
@@ -526,7 +526,8 @@
     "import type { BaseMessage } from \"@langchain/core/messages\";\n",
     "import { RunnablePassthrough, RunnableSequence } from \"@langchain/core/runnables\";\n",
     "\n",
-    "const filterMessages = ({ chat_history }: { chat_history: BaseMessage[]; }) => {\n",
+    "const filterMessages = (input: Record<string, unknown>) => {\n",
```

Comment:
Prefer `Record<string, any>`, not as type safe but for docs it makes things easier.
```suggestion
    "const filterMessages = (input: Record<string, any>) => {\n",
```